### DISCUSSION TITLE
Which build do i choose for my Ubuntu VM

### DISCUSSION THREAD

**Discussion by bttrfngrs** [simulot/immich-go#572] [General]
Kindly guide me on which build shall we use for our Ubuntu VM

**Comment by simulot** [comment]
It totally depends on your system. Could you tell more?

**Reply by bttrfngrs** [reply]
Thanks you for getting back simulot ! :) the 64 bit linux build worked like charm on my ubuntu VM.. gracias !

**Reply by simulot** [reply]
You're welcome
